qdrant-landing/content/documentation/cloud/create-cluster.md

```diff
@@ -10,24 +10,49 @@ This page shows you how to use the Qdrant Cloud Console to create a custom Qdran
 > **Prerequisite:** Please make sure you have provided billing information before creating a custom cluster. 
 
 1. Start in the **Clusters** section of the [Cloud Dashboard](https://cloud.qdrant.io). 
-2. Select **Clusters** and then click **+ Create**.
-3. A window will open. Enter a cluster **Name**.
-4. Currently, you can deploy to AWS, GCP, or Azure. 
-5. Choose your data center region. If you have latency concerns or other topology-related requirements, [**let us know**](mailto:cloud@qdrant.io).
-6. Configure RAM size for each node (1GB to 64GB). 
-> Please read [**Capacity and Sizing**](../../cloud/capacity-sizing/) to make the right choice. If you need more capacity per node, [**let us know**](mailto:cloud@qdrant.io).
-7. Choose the number of CPUs per node (0.5 core to 16 cores). The max/min number of CPUs is coupled to the chosen RAM size. 
-8. Select the number of nodes you want the cluster to be deployed on. 
+1. Select **Clusters** and then click **+ Create**.
+1. In the **Create a cluster** screen select **Free** or **Standard**
+   For more information on a free cluster, see the [Cloud quickstart](/documentation/cloud/quickstart-cloud). The remaining steps assume you want a standard cluster.
+1. Select a provider. Currently, you can deploy to AWS, GCP, or Azure. 
```

Comment:
Should provider names be in full? Amazon Web Services (AWS), Google Cloud Platform (GCP)?

And also, should they be within href _blank?

Reply:
It's a good idea to have provider names in full, once per page, so I've added them [here](https://github.com/qdrant/landing_page/pull/595/files#diff-8dc2c0982a232200859b7981d9a1e638fd283ccbf3f12486b82f702f94531b25R18-R20).